lib/withData.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { ApolloProvider, getDataFromTree } from 'react-apollo';
+import Head from 'next/head';
+import initApollo from './initApollo';
+import initRedux from './initRedux';
+import { selectEvent } from '../src/actions/index';
+
+function getComponentDisplayName(Component) {
+  return Component.displayName || Component.name || 'Unknown';
+}
+
+export default ComposedComponent => {
+  return class WithData extends Component {
+    static displayName = `WithData(${getComponentDisplayName(ComposedComponent)})`;
+    static propTypes = {
```

Comment:
It's my mistake))) I think they should be in each componet with input params.They should be immediately visible to other devs reading the file, since they serve as documentation(what props can be passed to component).